src/components/home/carousel.tsx

```diff
@@ -0,0 +1,47 @@
+import { useRouter } from 'expo-router';
+import { FlatList } from 'react-native';
+
+import { useProducts } from '@/api/products/use-products';
+import { Text, TouchableOpacity } from '@/ui';
+
+import { CarouselItem } from './carousel-item';
+export const HorizontalCarousel = () => {
+  const router = useRouter();
+  const {
+    data: products,
+    hasNextPage,
+    isFetchingNextPage,
+    fetchNextPage,
+  } = useProducts({ variables: { items: 7, text: '' } });
+  const productsToDisplay = products?.pages.flatMap((page) => page.data) || [];
+  const handleLoadMore = () => {
+    if (hasNextPage && !isFetchingNextPage) {
+      fetchNextPage();
+    }
+  };
+  return (
+    <>
+      <FlatList
+        className="h-[280]"
+        data={productsToDisplay}
+        contentContainerStyle={{
+          alignItems: 'center',
+          gap: 16,
+          paddingHorizontal: 16,
+        }}
```

Comment:
I think you can use the tailwind properties here


```suggestion
        contentContainerStyle="gap-4 px-4 items-center"
```